### PR TITLE
Keras callbacks

### PR DESCRIPTION
**Pull request by jvpoulos**
In ` Lrnr_lstm_keras` and ` Lrnr_gru_keras`, need to specify in fit() that callbacks is an element within args rather than an object. Without this change, the following error occurs when attempting to train either learner:

> Error in normalize_callbacks_with_metrics(view_metrics, initial_epoch, : object 'callbacks' not found

For early stopping to work, we need to specify a validation set. The easiest way is to include a validation_split argument in fit(). The default of 0 would be no validation set.

### CODE REVIEW

**Comment by nhejazi** [comment]
thanks for contributing these changes @jvpoulos, and sorry for the delay in reviewing. @rachaelvp my suggestion would be to merge into `devel` and then fix on the backend (updating the actions file and removing `devtools` from `Suggests`); happy to help with this.
